Comanche055/R31.agc

```diff
@@ -217,7 +217,7 @@ HAVEBASE	BON	RTB		# SUBSEQUENT INTEGRATIONS
 			RATT
 		STOVL	RONE
 			VATT
-		STODL	VONE		# GET SET FOR CONIC EXTRAP., OTHER.
```

Comment:
✔️ Removal of `.`

✨ Can you remove the space before `OTHER` 

✨ Can you remove the space between `LAST` and `---` above on L215